libr/bin/p/bin_mach0.c

```diff
@@ -288,6 +293,14 @@ static RList* symbols(RBinFile *bf) {
 	}
 	bool isStripped = false;
 	wordsize = MACH0_(get_bits) (obj->bin_obj);
+
+#if R_BIN_USE_PULL
```

Comment:
are you still setting this R_BIN_USE_PULL var? If you are, please just remove this #if and delete the OLD CODE below. If you are not setting this, just delete this part of the code.